packages/assets-controllers/src/NftController.ts

```diff
@@ -222,41 +218,235 @@ const controllerName = 'NftController';
 /**
  * The external actions available to the {@link NftController}.
  */
-type AllowedActions =
+export type AllowedActions =
   | AddApprovalRequest
```

Comment:
Like the `stateChange` event, the `getState` action is included by default in V2 controllers, and should always be defined (even if the state is empty).

```suggestion
export type NftControllerGetStateAction = ControllerGetStateAction<typeof controllerName, NftControllerState>;

export type NftControllerActions = NftControllerGetStateAction;

/**
 * The external actions available to the {@link NftController}.
 */
export type AllowedActions =
```